### ISSUE TITLE
UC - Right-of-Way Admin (50075) Process Update (Time of insert) (SR) (On PROD)

### ISSUE THREAD

**Issue by TracyLinder** [cityofaustin/atd-data-tech#5891]
### As UC Case Manager, I need to be able to use the Right-of-Way Admin (50075) Process prior to folder issuance. 

**Stakeholders:** Greg P, Isiah L, Jessica Davis, Jaclyn, Darren

**Acceptance Criteria:**
- Applies to UC folder
- Right-of-Way Admin Process(50075) must be inserted when UC folder is created.  
- User should be able to make process attempt 'Create Permits' in Right-of-Way Admin Process regardless of folder status
- When 'Create Permits' Attempt is made in ROW Admin Process, the following actions need to happen: 
-- Create child EX permit(s) based on Property records added to UC folder. (this works currently) 
-- The newly created Child permits need to be in status 'Pending Customer Action' (Pending Customer Action status needs to be added to EX folder)

**Comment by TracyLinder** [comment]
Future State Workflow:

**First (Stop-Gap)**

- Insert the ROW Admin at the beginning of the folder creation
- Create the EX child folders in Pending Customer Action

**Final (For CIP)**
- Upon activation of UC, the system will create the relative EX permit(s) w/ Pending Customer Action Status
- Sends automated emails all UC folder people records with the child folder information & help text

**Comment by TracyLinder** [comment]
**Update**
Will finish AC during next sprint, beginning 6/16
Will pull on deck the sprint after, beginning 6/30

FYI @Nadin-Nader & @vschandramouli

**Comment by TracyLinder** [comment]
@vschandramouli would you mind adding a comment with an update about the upcoming scoping meeting please?

**Comment by vschandramouli** [comment]
@TracyLinder Sure. 

Scheduled a scoping meeting with Greg and Jessica on 6/25. Will update AC when scoping is complete.

**Comment by vschandramouli** [comment]
@Nadin-Nader Updated AC for this ticket. It is ready for Backlog. There does seem to be one outstanding point from the original ticket description. I have pasted it below. Could you please refer and confirm if any other action is needed?

- We may need another issue for defining what the EX folder does after it's created as a child. 
    - See Unisys' work on ROWMAN Phase ?

**Comment by TracyLinder** [comment]
@Nadin-Nader can you assign Sree or Subha to this and add an estimate please?

**Comment by Sreelatha-BlazeKannan** [comment]
**Status **
@Nadin-Nader , @TracyLinder - Analysis is in progress

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader , @TracyLinder - Due to other cards updates, could not work on this till now; I will complete my analysis and development from today.

**Comment by Nadin-Nader** [comment]
@Sreelatha-BlazeKannan can you update this when you get a chance please

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader , @TracyLinder - Development completed. Testing and demo needs to be sent.

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader , @TracyLinder  - Tested this in Local host. please find the demo; if everything looks okay then this can be promoted to DEV.
https://user-images.githubusercontent.com/44907715/127928855-0ec9bfb8-8f1e-4891-b117-079d4bb3fb7b.mp4

**Comment by Nadin-Nader** [comment]
@Sreelatha-BlazeKannan this looks good to take it to TEST.

## **Approved for deployment to TEST🙌‼**

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader , @TracyLinder - DEV Deployment plan sent and here is the commit id: ac39d322124e for your review and approval

**Comment by Nadin-Nader** [comment]
@sree this has been sent to the DevOps CM Team for deployment to DEV under **SCTASK0159858**

**Comment by Nadin-Nader** [comment]
@Sreelatha-BlazeKannan i have approved the audit

**Comment by Nadin-Nader** [comment]
**Status:**
Waiting on DevOps to deploy to DEV under **SCTASK0163658**

**Comment by Nadin-Nader** [comment]
@Sreelatha-BlazeKannan this has been deployed to DEV. Please UAT

**Comment by Nadin-Nader** [comment]
This Never made it to V7 as it stopped in DEV due to FY22 Fees.
The last commit was 4ec28fd093fb. If we need the code, i can look it up in the old Phabricator for V6

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader , @TracyLinder - Analysis is in progress.

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader , @TracyLinder - Due to Other card priorities, this was started and put it on hold; Since I worked on this card in August 2021, I would like to continue and complete this card. Now I started to work on this again.

**Comment by TracyLinder** [comment]
Sree will be submitting the Dev DP today.

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader , @TracyLinder - Development Completed and Testing completed in Localhost; Submitted DEV DP of #5891 for your reference; Kindly approve and create task id for me.

Commit ID - [802da24a4d19](http://coaamandadevops02.austintexas.gov/R5:802da24a4d19a660dee19a4eae0353485d6b556a)

**Comment by Nadin-Nader** [comment]
@Sreelatha-BlazeKannan  this been sent to the DevOps Team for deployment to DEV under **SCTASK0279934**

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader , @TracyLinder - DEV Testing done, new changes are working as expected; however one of the existing functionality in dev is not working. Due to which unable to complete my testing and move forward. I also tested the existing functionality in TEST but it is not working and behaving like DEV. Now, Localhost existing functionality is working but not in DEV/TEST. So, I would need a meeting with business users to resolve this.

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader , @TracyLinder - Had a meeting and resolved the issue and showcased/demoed as per Acceptance Criteria. Now it is accepted for TEST & PROD Move. I will send the TEST DP soon.

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader , @TracyLinder -TEST DP of #5891 submitted. Kindly approve and create task id for your reference. 

Commit ID: 802da24a4d19

**Comment by Nadin-Nader** [comment]
@Sreelatha-BlazeKannan this has been deployed to TEST please UAT

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader , @TracyLinder - Tested in TEST Environment all working as expected. here are the sample folderrsn's for reference -  12878729/12878726. Will be sending the PROD DP by eod.

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader , @TracyLinder - PROD DP of #5891 submitted; Kindly approve and create task id for me.

Commit ID : 802da24a4d19

**Comment by Nadin-Nader** [comment]
This has been sent to the DevOps CM Team to be deployed to PROD under **SCTASK0286852**

**Comment by Nadin-Nader** [comment]
@Sreelatha-BlazeKannan this has been deployed to PROD. I will let Darrens team know via email

**Comment by Nadin-Nader** [comment]
Email sent to Greg P, Isiah L, Jessica Davis, Jaclyn, Darren informing them that this is on PROD and will be closed on Wednesday 2-22-23 if i do not hear back from them.
